test/api/v3/unit/libs/analyticsService.test.js

```diff
@@ -279,6 +279,38 @@ describe('analyticsService', () => {
           rewards: [{_id: 'reward'}],
           balance: 12,
           loginIncentives: 1,
+          background: {
+            blizzard: true,
+            blue: true,
+          },
+          shirt: {
+            horizon: true,
+          },
+          hair: {
+            base: {
+              1: true,
+            },
+            beard: {
+              3: true,
+            },
+            color: {
+              snowy: true,
+            },
+            mustache: {
+              2: true,
+            },
+          },
```

Comment:
These are set up on the wrong level of the user object. They should be under `user.purchased`